src/presentation/http/router/note.ts

```diff
@@ -154,11 +164,18 @@ const NoteRouter: FastifyPluginCallback<NoteRouterOptions> = (fastify, opts, don
     /**
      * TODO: Validate request query
      */
-    const { content } = request.body;
+    const content = request.body.content as JSON;
+    const parentPublicId = request.params.parentId;
     const { userId } = request;
 
```

Comment:
a lot of logic in the route, maybe inject `noteRelationshipService` to `noteService` instance?  Ideally, a route should call only one method